### DISCUSSION TITLE
Public server

### DISCUSSION THREAD

**Discussion by wyongjian** [rustdesk/rustdesk#4298] [General]
### Description

First，thanks the greatest project。I have a idle Oracle free  OCI，I want to contribute it as a public server。How I do？

### Impact

Public server

**Comment by rustdesk** [comment]
relay server requires a lot of bandwidth. What's your bandwidth?

**Reply by rustdesk** [reply]
If the bandwidth is ok, I need ssh access to your server (for ensuring the server can work robustly)

**Reply by wyongjian** [reply]
<img width="1053" alt="Snipaste_2023-05-08_11-36-53" src="https://user-images.githubusercontent.com/19405120/236728369-6ca1d5d7-806d-4769-ac44-a261a14125a9.png">

This is the Oracle Free VPS，I used one this is idle。

**Reply by rustdesk** [reply]
10 Mbps is low

**Reply by rustdesk** [reply]
![image](https://user-images.githubusercontent.com/71636191/236730100-79bca2ac-f2a5-42fd-aa6d-f8d527b59e31.png)

**Reply by wyongjian** [reply]
OK，If I have other resources available and will  contact you again。Thanks you shared project。

**Reply by rustdesk** [reply]
Which area is your server located?

**Reply by wyongjian** [reply]
Tokyo
